yt_dlp/extractor/wistia.py

```diff
@@ -399,7 +408,7 @@ def _real_extract(self, url):
 
         entries = [
             self.url_result(f'wistia:{video["hashedId"]}', WistiaIE, title=video.get('name'))
-            for video in traverse_obj(series, ('sections', ..., 'videos', ...)) or []
+            for video in traverse_obj(series, ('sections', ..., 'episodes', ...)) or []
```

Comment:
can we make this backwards compatible
```suggestion
            for video in traverse_obj(series, ('sections', ..., ('videos', 'episodes'), ...)) or []
```